simulator/src/test/java/com/hazelcast/simulator/worker/WorkerMessageProcessorTest.java

```diff
@@ -68,13 +72,13 @@ public void testMessageToRandomTest() throws Throwable {
         verifyMessageSentToEitherOr(testContainerMock1, testContainerMock2, message);
     }
 
-    private void verifyMessageSentToEitherOr(TestContainer<?> container1, TestContainer<?> container2, Message message) throws Throwable {
+    private void verifyMessageSentToEitherOr(TestContainer<?> container1, TestContainer<?> container2, Message message)
+            throws Throwable {
         try {
             verify(container1, timeout(TIMEOUT)).sendMessage(message);
-            verify(container2, timeout(TIMEOUT).never()).sendMessage(message);
-        } catch (WantedButNotInvoked e) { //the message was not deliver to container1
-            verify(container2).sendMessage(message); //so it should go to container2
+        } catch (WantedButNotInvoked e) {
```

Comment:
Since I updated `Mockito` this verify failed with a nicely "reminder exception" that `timeout()` and `never()` don't work together, although the syntax is compatible (some concurrency problems). So I changed these lines accordingly to the message of the exception.